docs/migrate.md

```diff
@@ -4,6 +4,39 @@ This document records manual migrations that are necessary to apply when upgradi
 Sourcegraph versions. All manual migrations between the version you are upgrading from and the
 version you are upgrading to should be applied (unless otherwise noted).
 
+## 3.14 (Unreleased)
+
+Version 3.14 changes the security context of the installation by switching to a non-root user for all containers.
+This allows running Sourcegraph in clusters with restrictive security policies.
+
+Existing installations that have been run as root before need to migrate their persistent volumes to work in 3.14.
```

Comment:
@uwe this needs to be called out in the CHANGELOG on sourcegraph/sourcegraph. Since if I am not mistaken if an admin doesn't do this, everything will implode :)

Reply:
done